app/upgrades/v19/upgrades.go

```diff
@@ -51,3 +59,23 @@ func RemoveOutpostsFromEvmParams(ctx sdk.Context,
 	params.ActivePrecompiles = newActivePrecompiles
 	return evmKeeper.SetParams(ctx, params)
 }
+
+// MigrateEthAccountsToBaseAccounts is used to store the code hash of the associated
+// smart contracts in the dedicated store in the EVM module and convert the former
+// EthAccounts to standard Cosmos SDK accounts.
+func MigrateEthAccountsToBaseAccounts(ctx sdk.Context, ak authkeeper.AccountKeeper, ek *evmkeeper.Keeper) {
+	ak.IterateAccounts(ctx, func(account authtypes.AccountI) (stop bool) {
+		ethAcc, ok := account.(*evmostypes.EthAccount)
+		if !ok {
+			return false
+		}
+
+		hexAddr := utils.CosmosToEthAddr(ethAcc.GetAddress())
```

Comment:
you don't need to convert anything. The address bytes are the same for cosmos and ethereum formats

Reply:
All functions in the `statedb` implementation take in a `common.Address`, so does this one - in order not to convert, I'd have to change the implementation to take in the `[]byte` instead. Either we have to change all functions or we keep it like this, IMO it makes no sense to have different signatures for the different methods in the same package.

Reply:
ok this is fine then